lib/font-face/index.js

```diff
@@ -1,11 +1,11 @@
 var attributes = require('./attributes.js');
 var font = require('../font');
-var extend = require('util')._extend;
+var assign = require('object-assign');
```

Comment:
Alright. I'll just make a mental note to remove this dependency once 0.12 becomes obsolete since `Object.assign` is built into 4.x+